realm/realm-library/src/main/java/io/realm/internal/RealmProxyMediator.java

```diff
@@ -138,6 +138,17 @@
      */
     public abstract <E extends RealmObject> E createDetachedCopy(E realmObject, int maxDepth, Map<RealmObject, RealmObjectProxy.CacheData<RealmObject>> cache);
 
+    /**
+     * Returns whether Realm transformer has been applied or not. Realm transformer adds overriding method
```

Comment:
2nd sentance, perhaps this instead: "Subclasses of this class are created by the annotation processor and the Realm transformer will add an overirde of this method that always return {@code true} if the transform was successfull."